lib/routes/enterprisecraftsmanship/index.ts

```diff
@@ -0,0 +1,74 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import logger from '@/utils/logger';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/archives',
+    categories: ['blog'],
+    example: '/enterprisecraftsmanship/archives',
+    radar: [
+        {
+            source: ['enterprisecraftsmanship.com/archives/'],
+        },
+    ],
+    url: 'enterprisecraftsmanship.com/',
+    name: 'Enterprise Craftsmanship',
+    maintainers: ['liyaozhong'],
+    handler,
+    description: 'Enterprise Craftsmanship blog archives',
+};
+
+async function handler() {
+    const rootUrl = 'https://enterprisecraftsmanship.com';
+    const currentUrl = `${rootUrl}/archives`;
+
+    const response = await got({
```

Comment:
Use either https://github.com/DIYgod/RSSHub/blob/5c33525c5facf7c450c65f08a2f66249ec69845f/lib/routes/enterprisecraftsmanship/index.ts#L28-L31 or https://github.com/DIYgod/RSSHub/blob/5c33525c5facf7c450c65f08a2f66249ec69845f/lib/routes/enterprisecraftsmanship/index.ts#L55 but not both style